src/tvhlog.c

```diff
@@ -144,7 +144,7 @@ tvhlog_subsys_t tvhlog_subsystems[] = {
   [LS_DESCRAMBLER]   = { "descrambler",   N_("Descrambler") },
   [LS_DESCRAMBLER_EMM]={ "descrambler-emm",N_("Descrambler EMM") },
   [LS_CACLIENT]      = { "caclient",      N_("CA (descrambling) Client") },
-  [LS_CSA]           = { "csa",           N_("CSA (descrambling)") },
+  [LS_TVHCSA]        = { "tvhcsa",        N_("TVHCSA (descrambling)") },
```

Comment:
I will reverse it, but I think even tvhcsa needs to be renamed. Originally it only did CSA descrambling, but now it includes DES and AES.

Reply:
Ohh, I got the point now. Perhaps, we can find another name (CSA means 'common scrambling algorithm' but this code in tvh is  'scrambling decoder' or so.. So SCRDEC or so might be more nice.

Reply:
More complete: CASCRDEC (Conditional Access SCRambling DECoder)

Reply:
"scrambling/descrambling" is a carry over from old analog TV where signals were inverted/modified so the picture would not sync and thus the picture was all "scrambled'. Since there is already a descrambler, maybe something related to decryption - since that is what is done there?

Reply:
tvh_decrypt?

Reply:
cadecrypt ?